srcpkgs/phonon-qt6/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'phonon-qt6'
+pkgname=phonon-qt6
+version=4.12.0
+revision=1
+build_style=cmake
+configure_args="-DPHONON_BUILD_QT5=OFF -DPHONON_INSTALL_QT_EXTENSIONS_INTO_SYSTEM_QT=ON"
+hostmakedepends="extra-cmake-modules qt6-tools qt6-base"
+makedepends="glib-devel qt6-base-devel qt6-tools-devel qt6-qt5compat-devel pulseaudio-devel"
+short_desc="Multimedia framework for KDE"
+maintainer="John <me@johnnynator.dev>"
```

Comment:
@Luciogi Are you @Johnnynator? If not, making someone else the maintainer of your template requires their consent.